App.js

```diff
@@ -18,14 +16,20 @@ import "firebase/firestore";
 // Imports necessary for redux
 import { Provider } from "react-redux";
 import store from "./redux/store.js";
+import { AppLoading } from "expo";
+import {
+  useFonts,
+  Inter_600SemiBold,
+  Inter_400Regular,
+} from "@expo-google-fonts/inter";
 
 // TODO: Replace the following with your app's Firebase project configuration
 const firebaseConfig = {
   apiKey: "AIzaSyDqW-IHyiK__ePv6T9JJDky6D1Vgj42iTM",
   authDomain: "fitness-tailor.firebaseapp.com",
   databaseURL: "https://fitness-tailor.firebaseio.com",
   projectId: "fitness-tailor",
-  storageBucket: "fitness-tailor.appspot.com",
+  storageBucket: "gs://fitness-tailor.appspot.com/",
```

Comment:
?

Reply:
I forgot to remove it. But If prof pic can be fetched with just `fitness-tailor.appspot.com`, let's change it back as it was